dask_sql/server/responses.py

```diff
@@ -131,10 +131,11 @@ def __init__(self, error: Exception):
         self.errorName = str(type(error))
         self.errorType = "USER_ERROR"
 
-        try:
-            self.errorLocation = {
-                "lineNumber": error.from_line + 1,
-                "columnNumber": error.from_col + 1,
-            }
-        except AttributeError:  # pragma: no cover
-            pass
+        # FIXME: ParserErrors currently don't contain information on where the syntax error occurred
```

Comment:
Yeah this is something that will require changes far up the third party stream in SqlParser-RS and then later in DataFusion. We can address this later however.

Reply:
cc @sarahyurick in case you have a minute to look into that, given your other work on the parser side of things